README.md

```diff
@@ -1,17 +1,19 @@
 # Profiling
-Profiling for Shopware
+Profiling for Shopware  
 Required Minimum Shopware Version 5.2
 
 # Installation
+
+## Zip Installation package for the Shopware Plugin Manager
+
+* Download the [latest plugin version](https://github.com/shyim/shopware-profiler/releases/latest/) (e.g. `ShyimProfiler-1.0.5.zip`)
```

Comment:
Unfortunately there is no way to link directly to the latest release package zip.